lib/trino-hive-formats/src/test/java/io/trino/hive/formats/avro/TestAvroPageDataReaderWithAvroNativeTypeManagement.java

```diff
@@ -244,14 +246,16 @@ public void testWriting()
             throws IOException, AvroTypeException
     {
         Location testLocation = createLocalTempLocation();
+        Type type = new NativeLogicalTypesAvroTypeBlockHandler().typeFor(ALL_SUPPORTED_TYPES_SCHEMA);
         try (AvroFileWriter fileWriter = new AvroFileWriter(
                 trinoLocalFilesystem.newOutputFile(testLocation).create(),
                 ALL_SUPPORTED_TYPES_SCHEMA,
                 new NativeLogicalTypesAvroTypeManager(),
                 AvroCompressionKind.NULL,
                 ImmutableMap.of(),
                 ALL_SUPPORTED_TYPES_SCHEMA.getFields().stream().map(Schema.Field::name).collect(toImmutableList()),
-                new NativeLogicalTypesAvroTypeBlockHandler().typeFor(ALL_SUPPORTED_TYPES_SCHEMA).getTypeParameters(), false)) {
+                type instanceof RowType rowType ? rowType.getFieldTypes() : List.of(),
```

Comment:
the `?` is weird. the `type` is effectively constant, so it's always either row or not (i suppose it's always a row)

Reply:
Ah, yes. I think I replicated that from another case where the type varies. Fixing.